docs_website/docs/changelog/breaking_change.md

```diff
@@ -7,12 +7,21 @@ slug: /changelog
 
 Here are the list of breaking changes that you should be aware of when updating Querybook:
 
+## v3.29.0
+
+Made below changes for `S3BaseExporter` (csv table uploader feature):
+
+-   Both `s3_path` and `use_schema_location` are optional now
```

Comment:
you need to mention if `use_schema_location=True` before 3.29.0, it would create managed table, now it creates external